mitodl/rapid-response-xblock
# edX Koa release compatibility

#### What are the relevant tickets?
- Fixes #85 
- Fixes https://github.com/mitodl/rapid-response-xblock/issues/73

NOTE: **This PR is rebased on other dependent PR https://github.com/mitodl/rapid-response-xblock/pull/86** (This rebase was required to pass all the tests for events)

#### What's this PR do?
- This PR refactors CapaModule changes and all other tests that were failing because of new `edx-platform` version upgrade (Koa in this case)
- Pins the Django version in settings.py, Same version that's used in the latest edx-plaform
- Fixes the warnings for Rapid-response & suppresses the warnings that were being generated through `edx-platform` (No direct fix except we make the changes in edx-platform for that or - we can rebase our master `mitodl/edx-platform` and test).

#### How should this be manually tested?
Majorly it should just be tested by having a look at the CI logs and making sure that the build passes all the checks.
Since it's rebased on https://github.com/mitodl/rapid-response-xblock/pull/86 and if that PR is already tested with local rapid response integration with `edx-platform` we should be good to go on this part.

#### Where should the reviewer start?
- Checking the CI

#### Any background context you want to provide?
Tests were failing on this repo over the time when edx probably made the transition to juniper. This PR makes the required refactoring w.r.t `edx-platform` Koa and gets the tests to pass.
